app/services/finance/declaration_statement_attacher.rb

```diff
@@ -5,16 +5,13 @@ module Finance
   # Work out which statement it should belong to
   # Attach the declaration to said statement
   class DeclarationStatementAttacher
-    attr_reader :participant_declaration
-
-    def initialize(participant_declaration:)
-      @participant_declaration = participant_declaration
+    def initialize(participant_declaration)
+      self.participant_declaration = participant_declaration
     end
 
     def call
       ApplicationRecord.transaction do
-        StatementLineItem.create!(
-          statement:,
+        statement.statement_line_items.create_or_find_by(
```

Comment:
The statement should always exist we've already seeded it until 2024. So I think it is fine. Actually a 500 is probably the more accurate response, since the lead provider params might be valid. So I think this is the correct behaviour